Examples/SubmissionFilterExample/Program.cs

```diff
@@ -0,0 +1,48 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using IndicoV2;
+using IndicoV2.Submissions.Models;
+
+namespace Examples
+{
+    public class SubmissionFilterExample
+    {
+        private static string GetToken() =>
+            File.ReadAllText(Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.UserProfile),
+                "indico_api_token.txt"));
+
+        public static async Task Main()
+        {
+            var filters = new OrFilter
+            {
+                Or = new List<IFilter>
+                {
```

Comment:
Could add a link in here to https://developer.indicodata.ai/docs/filter-submissions :).

Reply:
done! also made edits to C# example in readme: https://dash.readme.com/project/indicodata/v1.0/suggested/update/65ccfd81b338b90030082ebb

If you think it looks good I can merge it in as well